src/WikiService.Models.Dto/WikiService.Models.Dto.csproj

```diff
@@ -2,4 +2,8 @@
 
 	<Import Project="$(ProjectDir)..\..\Common.Configurations.targets" />
 
+	<ItemGroup>
+	  <PackageReference Update="LT.DigitalOffice.Kernel" Version="1.1.61" />
```

Comment:
in targets

Reply:
When I use version 1.1.63, I get error in stranger code (in EditArticleCommand).